api/src/main/java/org/openmrs/Cohort.java

```diff
@@ -348,13 +370,13 @@ public void setMemberIds(Set<Integer> memberIds) {
 		}
 	}
 	
-	public void setMemberships(Collection<CohortMembership> members) {
+	public void setMemberships(Set<CohortMembership> members) {
```

Comment:
This changes the method signature and it could cause errors in modules that use this method.

https://github.com/openmrs/openmrs-core/commit/60f3a26f15be46956f16cb1751737b3823476057

Reply:
Made the change. Could you please check this?